### PR TITLE
Fixed access code webservice issue with proguard

### PR DESCRIPTION
**Pull request by Yinten**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -112,22 +112,28 @@ android {
...
         debug {
+
             applicationIdSuffix ".debug"
             versionNameSuffix "-debug"
+
+
             buildConfigField "String", "BASE_URL_V2", "\"https://api.stage.godtoolsapp.com/godtools-api/rest/v2/\""
             buildConfigField "String", "BASE_URL", "\"https://api.stage.godtoolsapp.com/godtools-api/rest/\""
-            buildConfigField "String", "NEW_RELIC_API_KEY", "\"AA2f0cb2d41f6ac7dcc256eaa9ca7e0f640859fd52\""
+            buildConfigField "String", "NEW_RELIC_API_KEY", "\"AAcaf7bbfe3ffda5dcb1455d3bc16b0c5da1cd71e9\""
```
Comment (frett) [inline build.gradle]: the `NEW_RELIC_API_KEY`s got switched, they should be put back

**Comment by Yinten** [comment]
Oh snap OK

**Comment by Yinten** [comment]
Good eye, I've updated this.

**Comment by Yinten** [comment]
It did end up being this little bugger 


 +-keep public interface org.keynote.godtools.android.api.GodToolsApi

**Comment by Yinten** [comment]
These toasts don't show, wrong context or something.   I'll repair this in the net I/O refactor.

      else {
                authFailed(true, false);
            }

**Pull request merged**